### ISSUE TITLE
New submissions for Wed, 25 Jan 23

### ISSUE THREAD

**Issue by lizhihao6** [lizhihao6/get-daily-arxiv-noti#64]
## Keyword: events
There is no result 
## Keyword: event camera
There is no result 
## Keyword: events camera
There is no result 
## Keyword: white balance
There is no result 
## Keyword: color contrast
There is no result 
## Keyword: AWB
There is no result 
## Keyword: ISP
There is no result 
## Keyword: image signal processing
There is no result 
## Keyword: image signal process
There is no result 
### PowerQuant: Automorphism Search for Non-Uniform Quantization
 - **Authors:** Edouard Yvinec, Arnaud Dapogny, Matthieu Cord, Kevin Bailly
 - **Subjects:** Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2301.09858
 - **Pdf link:** https://arxiv.org/pdf/2301.09858
 - **Abstract**
 Deep neural networks (DNNs) are nowadays ubiquitous in many domains such as computer vision. However, due to their high latency, the deployment of DNNs hinges on the development of compression techniques such as quantization which consists in lowering the number of bits used to encode the weights and activations. Growing concerns for privacy and security have motivated the development of data-free techniques, at the expanse of accuracy. In this paper, we identity the uniformity of the quantization operator as a limitation of existing approaches, and propose a data-free non-uniform method. More specifically, we argue that to be readily usable without dedicated hardware and implementation, non-uniform quantization shall not change the nature of the mathematical operations performed by the DNN. This leads to search among the continuous automorphisms of $(\mathbb{R}_+^*,\times)$, which boils down to the power functions defined by their exponent. To find this parameter, we propose to optimize the reconstruction error of each layer: in particular, we show that this procedure is locally convex and admits a unique solution. At inference time, we show that our approach, dubbed PowerQuant, only require simple modifications in the quantized DNN activation functions. As such, with only negligible overhead, it significantly outperforms existing methods in a variety of configurations.
### K-Planes: Explicit Radiance Fields in Space, Time, and Appearance
 - **Authors:** Sara Fridovich-Keil, Giacomo Meanti, Frederik Warburg, Benjamin Recht, Angjoo Kanazawa
 - **Subjects:** Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2301.10241
 - **Pdf link:** https://arxiv.org/pdf/2301.10241
 - **Abstract**
 We introduce k-planes, a white-box model for radiance fields in arbitrary dimensions. Our model uses d choose 2 planes to represent a d-dimensional scene, providing a seamless way to go from static (d=3) to dynamic (d=4) scenes. This planar factorization makes adding dimension-specific priors easy, e.g. temporal smoothness and multi-resolution spatial structure, and induces a natural decomposition of static and dynamic components of a scene. We use a linear feature decoder with a learned color basis that yields similar performance as a nonlinear black-box MLP decoder. Across a range of synthetic and real, static and dynamic, fixed and varying appearance scenes, k-planes yields competitive and often state-of-the-art reconstruction fidelity with low memory usage, achieving 1000x compression over a full 4D grid, and fast optimization with a pure PyTorch implementation. For video results and code, please see sarafridov.github.io/K-Planes.
## Keyword: RAW
There is no result 
## Keyword: raw image
There is no result
